packages/cli/test/unit/util/dev/builder-cache.test.ts

```diff
@@ -1,235 +0,0 @@
-    const buildersPkg = {
-      dependencies: {
-        'not-bundled-package': '0.0.9',
-      },
-    };
-    const result = filterPackage(
-      'not-bundled-package@0.0.1',
-      buildersPkg,
-      cliPkg
-    );
-    expect(result).toEqual(true);
-  });
-
-  it('should filter install not bundled stable, cached version', () => {
-    const buildersPkg = {
-      dependencies: {
-        'not-bundled-package': '0.0.1',
-      },
-    };
-    const result = filterPackage('not-bundled-package', buildersPkg, cliPkg);
-    expect(result).toEqual(true);
-  });
-
-  it('should filter install not bundled tagged, cached tagged', () => {
-    const buildersPkg = {
-      dependencies: {
-        'not-bundled-package': '16.9.0-alpha.0',
-      },
-    };
-    const result = filterPackage(
-      'not-bundled-package@alpha',
-      buildersPkg,
-      cliPkg
-    );
-    expect(result).toEqual(true);
-  });
-});
-
-describe('getBuildUtils', () => {
-  const tests: [string[], string][] = [
-    [['@vercel/static', '@vercel/node@canary'], 'canary'],
-    [['@vercel/static', '@vercel/node@0.7.4-canary.0'], 'canary'],
-    [['@vercel/static', '@vercel/node@0.8.0'], 'latest'],
-    [['@vercel/static', '@vercel/node'], 'latest'],
-    [['@vercel/static'], 'latest'],
-    [['@vercel/md@canary'], 'canary'],
-    [['custom-builder'], 'latest'],
-    [['custom-builder@canary'], 'canary'],
-    [['canary-bird'], 'latest'],
-    [['canary-bird@4.0.0'], 'latest'],
-    [['canary-bird@canary'], 'canary'],
-    [['@canary/bird'], 'latest'],
-    [['@canary/bird@0.1.0'], 'latest'],
-    [['@canary/bird@canary'], 'canary'],
-    [['https://example.com'], 'latest'],
-    [[''], 'latest'],
-  ];
-
-  for (const [input, expected] of tests) {
-    it(`should install "${expected}" with input ${JSON.stringify(
-      input
-    )}`, () => {
-      const result = getBuildUtils(input);
-      expect(result).toEqual(`@vercel/build-utils@${expected}`);
-    });
-  }
-});
-
-describe('isBundledBuilder', () => {
-  const cliPkg = {
-    dependencies: {
-      '@vercel/node': '0.0.1',
-    },
-  };
-
-  it('should not detect when dependency does not appear in CLI package.json', () => {
-    const parsed = npa('@vercel/node');
-    const result = isBundledBuilder(parsed, {});
-    expect(result).toEqual(false);
-  });
-
-  it('should detect "canary" tagged releases', () => {
-    const parsed = npa('@vercel/node@canary');
-    const result = isBundledBuilder(parsed, cliPkg);
-    expect(result).toEqual(true);
-  });
-
-  it('should detect "canary" versioned releases', () => {
-    const parsed = npa('@vercel/node@1.6.1-canary.0');
-    const result = isBundledBuilder(parsed, cliPkg);
-    expect(result).toEqual(true);
-  });
-
-  it('should detect latest releases', () => {
-    const parsed = npa('@vercel/node');
-    const result = isBundledBuilder(parsed, cliPkg);
-    expect(result).toEqual(true);
-  });
-
-  it('should detect "latest" tagged releases', () => {
```

Comment:
Do we have similar tests somewhere else?

If not, we should convert these test cases to run against the new code.

Reply:
Here's a follow-up that adds a few more cases that were being covered in these previous tests https://github.com/vercel/vercel/pull/8554.